### ISSUE TITLE
Constructing a sequential model using weights of a graph model

### ISSUE THREAD

**Issue by hadi-ds** [fchollet/keras#2753]
I have trained a regression Graph model that takes two inputs vectors and return a score. It has three hidden dense layers that are shared among inputs (using .add_shared_node), and eventually the output vectors of the third layer are merged via inner product ('dot') to produce the score. 

model.get_weights() is a list of length 6, including three pairs of 'weight matrix, bias vector' one for each layer, with proper shapes (dimensions). This is what I expect. 

What I am trying to do is to construct a sequential model using the same weights. This will be a network that takes in ONE input vector and return an encoded version of that (with length equal to output_dim of third layer). 

I tried the following:
- construct that sequential model with three hidden layers
- save weights of graph model: 
  - graph_model.save_weights(...)
- load those weights into sequential: 
  - seq_model.load_weights('graph_weight.h5')

but this didn't work:
_Exception: You are trying to load a weight file containing 10 layers into a model with 3 layers._

Basically, since graph model has different architecture, it is as if graph model has 10 layers while the sequential has only three (graph_model.layers is list of length 10).

Like I said, graph_model.get_weights() actually gives the 3 weights and 3 biases. Is there a way to load those into my three-layer sequential model?

I appreciate any answers,
Hadi

**Comment by hadi-ds** [comment]
I managed a way of doing this indirectly. I first tried selecting part of the graph_weight.h5 file that included the weights of relevant layers, but it didn't work. 

Eventually, I grabbed weights and biases for all layers (given by model.get_weights) as numpy arrays, and basically wrote my own backend function that: takes inputs, multiplies them by weights, adds bias, applies activation, deliver it to the next layer, and so on.

**Comment by joelthchao** [comment]
Once you get the weight in numpy arrary format, you can use weights argument to insert weight manually. For `Sequential` model, it may look like:
```python
model = Sequential()
model.add(Dense(..., weights=[weight_1, bias_1]))
model.add(Dense(..., weights=[weight_2, bias_2]))
model.add(Dense(..., weights=[weight_3, bias_3]))
```

**Comment by kevinmartinmauro** [comment]
Also having a similar problem.
